docs/design/.pages

```diff
@@ -1,29 +1,17 @@
 nav:
     - Design: index.md
     - Software architecture:
-        - Design:
-            - Overview: design/software_architecture
-            - caret_analyze: design/software_architecture/caret_analyze
-            - caret_trace: design/software_architecture/caret_trace
-            - tilde: design/software_architecture/tilde
+        - Design: software_architecture
     - Runtime processing:
-        - Design:
-            - Overview: design/runtime_processing
-            - Hook: design/runtime_processing/hook
-            - Tracepoint filtering: design/runtime_processing/tracepoint_filtering
+        - Design: runtime_processing
     - Processing trace data:
-        - Design:
-            - Overview: design/processing_trace_data
-            - Records: design/processing_trace_data/records
-            - Response time: design/processing_trace_data/response_time
+        - Design: processing_trace_data
     - Tracepoints:
-        - Design:
-            - Overview: design/trace_points
-            - Initialization tracepoints: design/trace_points/initialization_trace_points
-            - Runtime tracepoints: design/trace_points/runtime_trace_points
-            - Differences from original ROS: design/trace_points/diff
+        - Design: trace_points
     - Configuration:
-        - Design:
-            - Overview: design/configuration
-            - File format: design/configuration/architecture_file
+        - Design: configuration
+    - Latency definitions:
```

Comment:
```suggestion
    - Event and latency definition:
```

Reply:
@hsgwa  
Could you rename `latency_definitions` directory to `event_and_latency_definition`?